source/collections/time-series-collection.txt

```diff
@@ -52,18 +52,38 @@ Procedure
              is used to label a unique series of documents. 
 
          * - ``granularity``
-           - The ``granularity`` field allows specifying a coarser granularity so 
-             measurements over a longer time span can be more efficiently stored 
-             and queried.
+           - The ``granularity`` field allows you to specify a coarser 
+             granularity so measurements over a longer time span can be more 
+             efficiently stored and queried.
 
              The default value is set to ``seconds``.
 
+             If you set the ``granularity`` parameter, you can't set the 
+             ``bucketMaxSpanSeconds`` and ``bucketRoundingSeconds`` parameters.
+
          * - ``expireAfterSeconds``
            -  The ``expireAfterSeconds`` field enables automatic deletion of 
               documents older than the specified number of seconds.  
+         
+         * - ``bucketMaxSpanSeconds``
+           -  The ``bucketMaxSpanSeconds`` field specifies the maximum time span 
```

Comment:
[suggestion]

This issue was already present in the existing copy, but it feels a little clunky to me to lead each description with the name of the field we're describing. For example, I think the description for `bucketMaxSpanSeconds` would just begin with:

"Specifies the maximum time span..."

Slightly out of scope for this particular ticket, but something to consider while we're looking at this page.

Reply:
Agree with you on it reading a bit awkwardly with the descriptions starting with the option names. Not too major of a change, so I'll go ahead and do that in this PR.